<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TweetsSearch</class>
 <widget class="QWidget" name="TweetsSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>#Поиск</string>
  </property>
  <widget class="QLineEdit" name="searchField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="searchedResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="loadProgress">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Предыдущая страница</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Следующая страница</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
